package.json

```diff
@@ -44,7 +44,8 @@
   "resolutions": {
     "@types/react": "^17",
     "@types/react-dom": "^17",
-    "@mui/material": "5.16.4"
+    "@mui/material": "5.16.4",
+    "axios": "1.7.4"
```

Comment:
Det er ikke sikkert at du trenger å pinne axios til en versjon her. Det kan skape uforventet oppførsel fra pakker som forventer en annen versjon. Har du testet å kjøre `yarn upgrade axios` for å kun oppdatere lockfilen? Så kan du se hva du har fått med `yarn list --pattern axios`

https://classic.yarnpkg.com/lang/en/docs/cli/upgrade/

Reply:
TIL :+1:  PR er oppdatert